percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.24-25.11.rst

```diff
@@ -0,0 +1,58 @@
+
+ :variable:`wsrep_replicate_myisam` variable is now a session variable (:bug:`1280280`).
+
+ |Percona XtraDB Cluster| now uses ``getifaddrs`` for node address detection (:bug:`1252700`). 
+
+ |Percona XtraDB Cluster| has implemented two new status variables: :variable:`wsrep_cert_bucket_count` and :variable:`wsrep_gcache_pool_size` for better instrumentation of galera memory usage. :variable:`wsrep_cert_bucket_count` shows the number of cells in the certification index hash-table and :variable:`wsrep_gcache_pool_size` shows the size of the page pool and/or dynamic memory allocated for gcache (in bytes).
+
+Bugs fixed 
+==========
+
+ Using the Rolling Schema Upgrade as a schema upgrade method due to conflict with :variable:`wsrep_desync` would allows only one ``ALTER TABLE`` to run concurrently. Bugs fixed :bug:`1330944` and :bug:`1330941`.
+
+ SST would resume even when the donor was already detected as being in ``SYNCED`` state. Bug fixed :bug:`1288528`.
+
+ Bug fixed? :bug:`1401133`.
+
+ DDL would fail on a node when running a TOI DDL, if one of the nodes has the table locked. Bug fixed :bug:`1376747`.
+
+ Bug fixed :bug:`1402166`.
+
+ ``xinet.d`` mysqlchk file was missing ``type = UNLISTED`` to work out of the box. Bug fixed :bug:`1418614`.
+
+ Conflict between `enforce_storage_engine <http://www.percona.com/doc/percona-server/5.6/management/enforce_engine.html#enforce_storage_engine>`_ and :variable:`wsrep_replicate_myisam` for ``CREATE TABLE`` has been fixed. Bug fixed :bug:`1435482`.
+
+ A specific trigger execution on the master server could cause a slave assertion error under row-based replication. The trigger would satisfy the following conditions: 1) it sets a savepoint; 2) it declares a condition handler which releases this savepoint; 3) the trigger execution passes through the condition handler. Bug fixed :bug:`1438990`.
+
+ |Percona XtraDB Cluster| init script was testing connection with wrong credentials. Bug fixed :bug:`1439673`.
+
+ Bug fixed :bug:`1441762`.
+
+ ``garbd-systemd`` didn't read the configuration file in :file:`/etc/sysconfig/garb`. Bug fixed :bug:`1444007`.
+
+ SST will now fail when move-back fails instead of continuing and failing at the next step. Bug fixed :bug:`1451670`.
+
+ |Percona XtraDB Cluster| ``.deb`` binaries were built without fast mutexes. Bug fixed :bug:`1457118`.
+
+ The error message text returned to the client in the non-primary mode is now more descriptive (``"WSREP has not yet prepared node for application use"``), instead of ``"Unknown command"`` returned previously. Bug fixed :bug:`1426378`.
+
+ Out-of-bount memory access issue in ``seqno_reset()`` function has been fixed.
+
+ :variable:`wsrep_local_cached_downto` would underflow when the node on which it is queried has no writesets in gcache.
+
+ 
+Other bugs fixed: :bug:`1290526`.
```

Comment:
Yes, I've skipped some like: JEN-342, PXC-368, PXC-369, that were not affecting users